app/javascript/src/Applications/create_application.js

```diff
@@ -47,6 +47,14 @@ export class CreateApplication {
     return this.relationPlansServices[this.selectedPlan]
   }
 
+  get relationServiceAndCreateServicePlanPath (): {[number]: string} {
+    return this.metadata.data('relation_service_and_create_service_plan_path')
+  }
+
+  getCreateServicePlanPathForService (serviceId: number): string {
+    return this.relationServiceAndCreateServicePlanPath[serviceId]
```

Comment:
Something like this
in the view:

```ruby
data: {admin_service_service_plans_path: admin_service_service_plans_path(':service_id') }
```

And in the JS:
```javascript
return this.metadata.data('admin_service_service_plans_path').replace(':service_id', number)
```

Reply:
OK